src/app/js/i18n/translations/fr.js

```diff
@@ -224,4 +224,12 @@ export default {
     upload_url: 'upload from url',
     invalid_url: 'url is invalid',
     or: 'or',
+    html: 'Pour des balises HTML',
+    list: 'Pour une liste de valeurs',
+    uri: 'Pour un lien interne',
+    email: 'Pour un lien courriel',
+    image: 'Pour l\'URL d\'une image',
+    link: 'Pour une URL externe',
+    chart: 'Pour un graphique contextuel',
+    istex: 'Pour une requête ISTEX ',
```

Comment:
Would'nt it be better with `Balises HTML`, `URL externe`: shorter, better fit with English version?